auth0/provider.go

```diff
@@ -77,20 +95,28 @@ func Provider() *schema.Provider {
 }
 
 func Configure(data *schema.ResourceData) (interface{}, error) {
-
-	domain := data.Get("domain").(string)
-	id := data.Get("client_id").(string)
-	secret := data.Get("client_secret").(string)
```

Comment:
will bring them back, but they'll have to be in the if statement as the cast will cause the provider to panic if they're using the management token since client_id and client_secret will be nil